vscode/src/test/suite/ruby/shadowenv.test.ts

```diff
@@ -110,7 +118,9 @@ suite("Shadowenv", () => {
   });
 
   afterEach(() => {
-    fs.rmSync(rootPath, { recursive: true, force: true });
+    if (rootPath) {
+      fs.rmSync(rootPath, { recursive: true, force: true });
+    }
```

Comment:
`rootPath` is sometimes undefined.

Truthfully, I have no idea why :joy: Open to suggestions if anyone has any idea.

Reply:
I _suspect_ that may be related to using async exec. If you switch from `asyncExec` to execSync to check for Shadowenv, do you still see the issue?

Reply:
I think you were right on this.